docs/Project.toml

```diff
@@ -1,6 +1,10 @@
 [deps]
+Crystal = "3c6eccdf-2a89-4c24-a1d4-ff210daa8476"
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 Molly = "aa0f7f06-fcc0-5ec4-a7f3-a573f33f9c4c"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```

Comment:
It would indeed be better to use the same plotting package throughout, though at the minute the code in the docs is not run during the docs build and Makie plots are pasted as images into the doc files.

It would be good if the docs notebook appeared in the docs - perhaps it could go in the examples section as markdown. Then these docs dependencies can be removed until we deal with doctests later, as the examples can assume that users have the package installed.

Reply:
Tried using `Makie` before but had some problem I can't remember. `Plots` is mostly required for the `docs/fs.ipynb` notebook. Can try to replace `Plots` there though if required.